main.go

```diff
@@ -32,75 +46,87 @@ func main() {
 
 	switch os.Args[1] {
 	case "init":
-		initMageTools()
+		if err := initMageTools(); err != nil {
+			panic(err)
+		}
+	case "gen":
```

Comment:
@odsod thoughts right now i have not exposed `gen` in the help, meaning its "invisible" for the user, thoughts?

Reply:
no strong opinion here :+1: